client/onboarding/style.scss

```diff
@@ -65,6 +55,7 @@ body.wcpay-onboarding__body {
 
 			&__heading {
 				@include wp-title-large;
+				font-family: 'SF Pro Display', $default-font;
```

Comment:
I am not sure if we should change font-family, I didn't find any occurrences in the codebase like this

Reply:
The context for this is in this P2 post (paJDYF-ci2-p2), specifically:

> The designs are using SF Pro Display at 32px, which is a slight change from the existing SF Pro Text used for the headers. The nuance in letterspacing makes the headlines more readable for larger sizes.

I think it should hopefully be safe to do so, because as far as I can tell the only difference with the "Display" font is that it renders more smoothly for large font sizes. It'll just fall back to the default font set (which is SF Pro on OSX devices, and then falling back to equivalent fonts on other devices) if there's any problem.

Reply:
thanks for explaining! I'm good with that!